updated Readme (#383)
PediatricOpenTargets/OpenPBTA-analysis: analyses/README.md

> | [data-pre-release-qc](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/data-pre-release-qc)					| 'histologies.tsv' <br> 'gene-counts-rsem-expected_count-collapsed.rds' <br> 'gene-expression-rsem-tpm-collapased.rds' <br> 'tcga-gene-counts-rsem-expected_count-collapsed.rds' <br> 'tcga-gene-expression-rsem-tpm-collapsed.rds' <br> 'cnv-cnvkit.seg.gz' <br> 'cnvkit_with_status.tsv' <br> 'consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz' <br> 'consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz' <br> 'snv-mutation-tmb-all.tsv' <br> 'fusion_summary_embryonal_foi.tsv' <br> 'fusion_summary_ependymoma_foi.tsv' <br> 'fusion_summary_lgat_foi.tsv' <br> fusion_summary_ewings_foi.tsv' <br> 'biospecimen_id_to_bed_map.txt'					| Performs QC on data pre-release files with requirements which should pass before hand off between BIXU Engineering team to the OpenPedCan team		| Yes					| 'data-pre-release-qc.nb.html'						| N/A					| No		| N/A		| N/A | 

Reviewer: ```suggestion
| [data-pre-release-qc](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/data-pre-release-qc)					| `histologies.tsv` <br> `gene-counts-rsem-expected_count-collapsed.rds` <br> `gene-expression-rsem-tpm-collapased.rds` <br> `tcga-gene-counts-rsem-expected_count-collapsed.rds` <br> `tcga-gene-expression-rsem-tpm-collapsed.rds` <br> `cnv-cnvkit.seg.gz` <br> `cnvkit_with_status.tsv` <br> `consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz` <br> `consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz` <br> `snv-mutation-tmb-all.tsv` <br> `fusion_summary_embryonal_foi.tsv` <br> `fusion_summary_ependymoma_foi.tsv` <br> `fusion_summary_lgg_hgg_foi.tsv` <br> `fusion_summary_ewings_foi.tsv` <br> `biospecimen_id_to_bed_map.txt`					| Performs QC on data pre-release files with requirements which should pass before hand off between BIXU Engineering team to the OpenPedCan team		| Yes					| 'data-pre-release-qc.nb.html'						| N/A					| No		| N/A		| N/A | 
```

> | [focal-cn-file-preparation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/focal-cn-file-preparation)                         | `cnv-cnvkit.seg.gz` <br>`cnv-controlfreec.tsv.gz` <br>`gene-expression-rsem-tpm-collapsed.rds`<br> `cnv-consensus.seg.gz`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Maps from copy number variant caller segments to gene identifiers; will be updated to take into account changes that affect entire cytobands, chromosome arms [#186](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/186)                                                                                                                                                                                                                                                                                                                                                                                                        | Yes                              | `results/cnvkit_annotated_cn_wxs_autosomes.tsv.gz` <br>`results/cnvkit_annotated_cn_wxs_x_and_y.tsv.gz` <br> `results/consensus_seg_annotated_cn_autosomes.tsv.gz`  <br>`results/consensus_seg_annotated_cn_x_and_y.tsv.gz` `results/consensus_seg_most_focal_fn_status.tsv.gz` <br> `consensus_seg_recurrent_focal_cn_units.tsv` <br> consensus_seg_with_ucsc_cytoband_status.tsv.gz` <br> `results/consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz` included in data download <br>`results/consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz` included in data download                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Yes              | Yes              | CAVATICA                                                                                                    | N/A                          |

Reviewer: ```suggestion
| [focal-cn-file-preparation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/focal-cn-file-preparation)                         | `cnv-cnvkit.seg.gz` <br>`cnv-controlfreec.tsv.gz` <br>`gene-expression-rsem-tpm-collapsed.rds`<br> `cnv-consensus.seg.gz`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Maps from copy number variant caller segments to gene identifiers; will be updated to take into account changes that affect entire cytobands, chromosome arms [#186](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/186)                                                                                                                                                                                                                                                                                                                                                                                                        | Yes                              | `cnvkit_annotated_cn_wxs_autosomes.tsv.gz` <br>`cnvkit_annotated_cn_wxs_x_and_y.tsv.gz` <br> `consensus_seg_annotated_cn_autosomes.tsv.gz`  <br>`consensus_seg_annotated_cn_x_and_y.tsv.gz` `consensus_seg_most_focal_fn_status.tsv.gz` <br> `consensus_seg_recurrent_focal_cn_units.tsv` <br> consensus_seg_with_ucsc_cytoband_status.tsv.gz` <br> `consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz` included in data download <br>`consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz` included in data download                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Yes              | Yes              | CAVATICA                                                                                                    | N/A                          |
```

> | [gene_match](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/gene_match)                                                       | GTF file sources: [gencode v28](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_28/gencode.v28.annotation.gtf.gz) [gencode v38](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_38/gencode.v38.annotation.gtf.gz) `open_ped_can_v7_ensg-hugo-rmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | This module reads GTF file and formats `attributes` to extract gene symbol with gene ensembl ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Yes                              | `results\ensg-hugo-pmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Yes              | Yes              | GitHub                                                                                                      | N/A                          |

Reviewer: ```suggestion
| [gene_match](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/gene_match)                                                       | GTF file sources: [gencode v28](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_28/gencode.v28.annotation.gtf.gz) [gencode v38](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_38/gencode.v38.annotation.gtf.gz) `open_ped_can_v7_ensg-hugo-rmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | This module reads GTF file and formats `attributes` to extract gene symbol with gene ensembl ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Yes                              | `ensg-hugo-pmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Yes              | Yes              | GitHub                                                                                                      | N/A                          |
```

> | [gene_match](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/gene_match)                                                       | GTF file sources: [gencode v28](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_28/gencode.v28.annotation.gtf.gz) [gencode v38](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_38/gencode.v38.annotation.gtf.gz) `open_ped_can_v7_ensg-hugo-rmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | This module reads GTF file and formats `attributes` to extract gene symbol with gene ensembl ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Yes                              | `ensg-hugo-pmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Yes              | Yes              | GitHub                                                                                                      | N/A                          |

Reviewer: ```suggestion
| [gene_match](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/gene_match)                                                       | GTF file sources: [gencode v28](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_28/gencode.v28.annotation.gtf.gz) <br>[gencode v38](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_38/gencode.v38.annotation.gtf.gz) <br>`open_ped_can_v7_ensg-hugo-rmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | This module reads GTF file and formats `attributes` to extract gene symbol with gene ensembl ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Yes                              | `ensg-hugo-pmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Yes              | Yes              | GitHub                                                                                                      | N/A                          |
```

> | [gene_match](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/gene_match)                                                       | GTF file sources: [gencode v28](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_28/gencode.v28.annotation.gtf.gz) <br>[gencode v38](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_38/gencode.v38.annotation.gtf.gz) <br>`open_ped_can_v7_ensg-hugo-rmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | This module reads GTF file and formats `attributes` to extract gene symbol with gene ensembl ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Yes                              | `ensg-hugo-pmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Yes              | Yes              | GitHub                                                                                                      | N/A                          |

Reviewer: ```suggestion
| [gene_match](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/gene_match)                                                       | GTF file sources: <br> [gencode v28](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_28/gencode.v28.annotation.gtf.gz) <br>[gencode v38](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_38/gencode.v38.annotation.gtf.gz) <br>`open_ped_can_v7_ensg-hugo-rmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | This module reads GTF file and formats `attributes` to extract gene symbol with gene ensembl ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Yes                              | `ensg-hugo-pmtl-mapping.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Yes              | Yes              | GitHub                                                                                                      | N/A                          |
```

> | [immune-deconv](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/immune-deconv)                                                 | `gene-expression-rsem-tpm-collapsed.rds` <br> `data/histologies.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Immune/Stroma characterization across PBTA part of [#15](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/15)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | No                               | `results/xcell_output.rds` <br> `results/quantiseq_output.rds`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | No               | No               | N/A                                                                                                         | N/A                          |

Reviewer: ```suggestion
| [immune-deconv](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/immune-deconv)                                                 | `gene-expression-rsem-tpm-collapsed.rds` <br> `data/histologies.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Immune/Stroma characterization across PBTA part of [#15](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/15)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | No                               | `xcell_output.rds` <br> `quantiseq_output.rds`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | No               | No               | N/A                                                                                                         | N/A                          |
```

> | [methylation-preprocessing](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/methylation-preprocessing)                         | `metadata\TARGET_Normal_MethylationArray_20160812.sdrf.txt` <br>`metadata\TARGET_NBL_MethylationArray_20160812.sdrf.1.txt` <br>`TARGET_NBL_MethylationArray_20160812.sdrf.2.txt` <br>`metadata/TARGET_CCSK_MethylationArray_20160819.sdrf.txt` <br>`metadata\TARGET_OS_MethylationArray_20161103.sdrf.txt` <br>`metadata\TARGET_WT_MethylationArray_20160831.sdrf.txt` <br>`metadata\TARGET_AML_MethylationArray_20160812_450k.sdrf.1.txt` <br>`metadata\TARGET_AML_MethylationArray_20160812_450k.sdrf.2.txt` <br>`metadata\TARGET_AML_MethylationArray_20160812_27k.sdrf.1.txt` <br>`metadata\TARGET_AML_MethylationArray_20160812_27k.sdrf.2.txt` <br>`metadata\TARGET_AML_MethylationArray_20160812_27k.sdrf.3.txt` <br>`metadata\manifest_methylation_CBTN_20220410.1.csv` <br>`metadata\manifest_methylation_CBTN_20220410.2.csv` <br>`metadata\manifest_methylation_CBTN_20220410.3.csv` <br>`metadata\manifest_methylation_CBTN_20220410.4.csv` | Preprocess probe hybridization intensity values of selected methylated and unmethylated cytosine (CpG) loci into usable methylation measurements for the Pediatric Open Targets, OpenPedCan-analysis raw DNA methylation array datasets.                                                                                                                                                                                                                                                                                                                                  | No                               | N/A                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Yes              | Yes              | Cavatica                                                                                                      | N/A                          |

Reviewer: ```suggestion
| [methylation-preprocessing](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/methylation-preprocessing)                         | `TARGET_Normal_MethylationArray_20160812.sdrf.txt` <br>`TARGET_NBL_MethylationArray_20160812.sdrf.1.txt` <br>`TARGET_NBL_MethylationArray_20160812.sdrf.2.txt` <br>`TARGET_CCSK_MethylationArray_20160819.sdrf.txt` <br>`TARGET_OS_MethylationArray_20161103.sdrf.txt` <br>`TARGET_WT_MethylationArray_20160831.sdrf.txt` <br> `TARGET_AML_MethylationArray_20160812_450k.sdrf.1.txt` <br>`TARGET_AML_MethylationArray_20160812_450k.sdrf.2.txt` <br>`TARGET_AML_MethylationArray_20160812_27k.sdrf.1.txt` <br>`TARGET_AML_MethylationArray_20160812_27k.sdrf.2.txt` <br>`TARGET_AML_MethylationArray_20160812_27k.sdrf.3.txt` <br>`manifest_methylation_CBTN_20220410.1.csv` <br>`manifest_methylation_CBTN_20220410.2.csv` <br>`manifest_methylation_CBTN_20220410.3.csv` <br>`manifest_methylation_CBTN_20220410.4.csv` | Preprocess probe hybridization intensity values of selected methylated and unmethylated cytosine (CpG) loci into usable methylation measurements for the Pediatric Open Targets, OpenPedCan-analysis raw DNA methylation array datasets.                                                                                                                                                                                                                                                                                                                                  | No                               | N/A                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Yes              | Yes              | Cavatica                                                                                                      | N/A                          |
```

> | [filter-mtp-tables](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/filter-mtp-tables)       | `gencode.v39.primary_assembly.annotation.gtf.gz` <br>`PMTL_v1.1.tsv`<br> `histologies.tsv` <br>`gene-level-snv-consensus-annotated-mut-freq.tsv.gz` <br>`snv-consensus-plus-hotspots.maf.tsv.gz`<br> `variant-level-snv-consensus-annotated-mut-freq.tsv.gz` <br>`gene-level-cnv-consensus-annotated-mut-freq.tsv.gz` `consensus_wgs_plus_cnvkit_wxs.tsv.gz`<br> `putative-oncogene-fusion-freq.tsv.gz`<br> `fusion-putative-oncogenic.tsv` <br>`putative-oncogene-fused-gene-freq.tsv.gz` <br> `long_n_tpm_mean_sd_quantile_gene_wise_zscore.tsv.gz` <br> `long_n_tpm_mean_sd_quantile_group_wise_zscore.tsv.gz`                                                                                                                                                                                                                                                                                                                                                                                                                                       | Remove Ensembl (ESNG) gene identifier in the OPenPedCan mutation frequency tables, including SNV, CNV, fusion, and TPM expression tables that are not in GENCODE v39 and Ensembl package 104.                                                                                                                                                                                                                                                                                                                                                                                                     | No                               | All files from module `results` directory                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | No               | Yes              | N/A                                                                                                         | Yes                          |

Reviewer: ```suggestion
| [filter-mtp-tables](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/filter-mtp-tables)       | `gencode.v39.primary_assembly.annotation.gtf.gz` <br>`PMTL_v1.1.tsv`<br> `histologies.tsv` <br>`gene-level-snv-consensus-annotated-mut-freq.tsv.gz` <br>`snv-consensus-plus-hotspots.maf.tsv.gz`<br> `variant-level-snv-consensus-annotated-mut-freq.tsv.gz` <br>`gene-level-cnv-consensus-annotated-mut-freq.tsv.gz` `consensus_wgs_plus_cnvkit_wxs.tsv.gz`<br> `putative-oncogene-fusion-freq.tsv.gz`<br> `fusion-putative-oncogenic.tsv` <br>`putative-oncogene-fused-gene-freq.tsv.gz` <br> `long_n_tpm_mean_sd_quantile_gene_wise_zscore.tsv.gz` <br> `long_n_tpm_mean_sd_quantile_group_wise_zscore.tsv.gz`                                                                                                                                                                                                                                                                                                                                                                                                                                       | Remove Ensembl (ESNG) gene identifier in the OpenPedCan mutation frequency tables, including SNV, CNV, fusion, and TPM expression tables that are not in GENCODE v39 and Ensembl package 104.                                                                                                                                                                                                                                                                                                                                                                                                     | No                               | All files from module `results` directory                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | No               | Yes              | N/A                                                                                                         | Yes                          |
```

> | [focal-cn-file-preparation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/focal-cn-file-preparation)                         | `cnv-cnvkit.seg.gz` <br>`cnv-controlfreec.tsv.gz` <br>`gene-expression-rsem-tpm-collapsed.rds`<br> `cnv-consensus.seg.gz`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Maps from copy number variant caller segments to gene identifiers; will be updated to take into account changes that affect entire cytobands, chromosome arms [#186](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/186)                                                                                                                                                                                                                                                                                                                                                                                                        | Yes                              | `results/cnvkit_annotated_cn_wxs_autosomes.tsv.gz` `results/cnvkit_annotated_cn_wxs_x\_and_y.tsv.gz` `results/consensus_seg_annotated_cn_autosomes.tsv.gz` `results/consensus_seg_annotated_cn_x\_and_y.tsv.gz` `results/consensus_seg_most_)focal_fn_status.tsv.gz' 'consensus_seg_recurrent_focal_cn_units.tsv' ' consensus_seg_with_ucsc_cytoband_status.tsv.gz' ` `results/consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz` included in data download `results/consensus_wgs_plus_cnvkit_wxs_x\_and_y.tsv.gz` included in data download                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Yes              | Yes              | CAVATICA                                                                                                    | N/A                          |

Reviewer: ```suggestion
| [focal-cn-file-preparation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/focal-cn-file-preparation)                         | `cnv-cnvkit.seg.gz` <br>`cnv-controlfreec.tsv.gz` <br>`gene-expression-rsem-tpm-collapsed.rds`<br> `cnv-consensus.seg.gz`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Maps from copy number variant caller segments to gene identifiers; will be updated to take into account changes that affect entire cytobands, chromosome arms [#186](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/186)                                                                                                                                                                                                                                                                                                                                                                                                        | Yes                              | `results/cnvkit_annotated_cn_wxs_autosomes.tsv.gz` <br>`results/cnvkit_annotated_cn_wxs_x_and_y.tsv.gz` <br> `results/consensus_seg_annotated_cn_autosomes.tsv.gz`  <br>`results/consensus_seg_annotated_cn_x_and_y.tsv.gz` `results/consensus_seg_most_focal_fn_status.tsv.gz` <br> `consensus_seg_recurrent_focal_cn_units.tsv` <br> consensus_seg_with_ucsc_cytoband_status.tsv.gz` <br> `results/consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz` included in data download <br>`results/consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz` included in data download                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Yes              | Yes              | CAVATICA                                                                                                    | N/A                          |
```

> | [immune-deconv](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/immune-deconv)                                                 | `gene-expression-rsem-tpm-collapsed.rds` <br> 'data/histologies.tsv'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Immune/Stroma characterization across PBTA part of [#15](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/15)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | No                               | `results/xcell_output.rds` <br> `results/quantiseq_output.rds `                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | No               | No               | N/A                                                                                                         | N/A                          |

Reviewer: ```suggestion
| [immune-deconv](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/immune-deconv)                                                 | `gene-expression-rsem-tpm-collapsed.rds` <br> `data/histologies.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Immune/Stroma characterization across PBTA part of [#15](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/15)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | No                               | `results/xcell_output.rds` <br> `results/quantiseq_output.rds`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | No               | No               | N/A                                                                                                         | N/A                          |
```

| [methylation-summary](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/methylation-summary)				| 'infinium.gencode.v39.probe.annotations.tsv.gz' <br> 'independent-specimens.rnaseqpanel.eachchort.tsv' 'independent-specimens.methyl.eachcohort.tsv' 'gene-expression-rsem-tpm-collapsed.rds' 'rna-isoform-expression-rsem-tpm.rds' 'methyl-beta-values.rds' 'efo-mondo-map.tsv' 'histlogies.tsv'					| Summarize preprocessed Illumina Infinium Human Methylation array measurements produced by the OpenPedCan methylation preprocessing module and Illumina infinium methylation array CpG probe coordinates.					| No			|N/A				|No				| No				| aws			| N/A |

> 

Reviewer: ```suggestion
| [methylation-preprocessing](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/methylation-preprocessing)                         | `metadata\TARGET_Normal_MethylationArray_20160812.sdrf.txt` <br>`metadata\TARGET_NBL_MethylationArray_20160812.sdrf.1.txt` <br>`TARGET_NBL_MethylationArray_20160812.sdrf.2.txt` <br>`metadata/TARGET_CCSK_MethylationArray_20160819.sdrf.txt` <br>`metadata\TARGET_OS_MethylationArray_20161103.sdrf.txt` <br>`metadata\TARGET_WT_MethylationArray_20160831.sdrf.txt` <br>`metadata\TARGET_AML_MethylationArray_20160812_450k.sdrf.1.txt` <br>`metadata\TARGET_AML_MethylationArray_20160812_450k.sdrf.2.txt` <br>`metadata\TARGET_AML_MethylationArray_20160812_27k.sdrf.1.txt` <br>`metadata\TARGET_AML_MethylationArray_20160812_27k.sdrf.2.txt` <br>`metadata\TARGET_AML_MethylationArray_20160812_27k.sdrf.3.txt` <br>`metadata\manifest_methylation_CBTN_20220410.1.csv` <br>`metadata\manifest_methylation_CBTN_20220410.2.csv` <br>`metadata\manifest_methylation_CBTN_20220410.3.csv` <br>`metadata\manifest_methylation_CBTN_20220410.4.csv` | Preprocess probe hybridization intensity values of selected methylated and unmethylated cytosine (CpG) loci into usable methylation measurements for the Pediatric Open Targets, OpenPedCan-analysis raw DNA methylation array datasets.                                                                                                                                                                                                                                                                                                                                  | No                               | N/A                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Yes              | Yes              | Cavatica                                                                                                      | N/A                          |
| [methylation-summary](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/methylation-summary)				| `infinium.gencode.v39.probe.annotations.tsv.gz` <br> `independent-specimens.rnaseqpanel.eachchort.tsv` <br>`independent-specimens.methyl.eachcohort.tsv` <br> `gene-expression-rsem-tpm-collapsed.rds` <br> `rna-isoform-expression-rsem-tpm.rds` <br> `methyl-beta-values.rds` <br> `efo-mondo-map.tsv` <br> `histlogies.tsv`					| Summarize preprocessed Illumina Infinium Human Methylation array measurements produced by the OpenPedCan methylation preprocessing module and Illumina infinium methylation array CpG probe coordinates.					| No			| N/A				| No				| No				| aws			| N/A |
```

Reviewer: update some of those single quotes to backticks and remove line spaces

> 

Reviewer: ```suggestion
```
remove empty line

> | [data-pre-release-qc](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/data-pre-release-qc)					| 'histologies.tsv' <br> 'gene-counts-rsem-expected_count-collapsed.rds' <br> 'gene-expression-rsem-tpm-collapased.rds' <br> 'tcga-gene-counts-rsem-expected_count-collapsed.rds' <br> 'tcga-gene-expression-rsem-tpm-collapsed.rds' <br> 'cnv-cnvkit.seg.gz' <br> 'cnvkit_with_status.tsv' <br> 'consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz' <br> 'consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz' <br> 'snv-mutation-tmb-all.tsv' <br> 'fusion_summary_embryonal_foi.tsv' <br> 'fusion_summary_ependymoma_foi.tsv' <br> 'fusion_summary_lgat_foi.tsv' <br> fusion_summary_ewings_foi.tsv' <br> 'biospecimen_id_to_bed_map.txt'					| Performs QC on data pre-release files with requirements which should pass before hand off between BIXU Engineering team to the OpenPedCan team		| Yes					| 'data-pre-release-qc.nb.html'						|N/A					| No		| N/A		| N/A | 

Reviewer: ```suggestion
| [data-pre-release-qc](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/data-pre-release-qc)					| 'histologies.tsv' <br> 'gene-counts-rsem-expected_count-collapsed.rds' <br> 'gene-expression-rsem-tpm-collapased.rds' <br> 'tcga-gene-counts-rsem-expected_count-collapsed.rds' <br> 'tcga-gene-expression-rsem-tpm-collapsed.rds' <br> 'cnv-cnvkit.seg.gz' <br> 'cnvkit_with_status.tsv' <br> 'consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz' <br> 'consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz' <br> 'snv-mutation-tmb-all.tsv' <br> 'fusion_summary_embryonal_foi.tsv' <br> 'fusion_summary_ependymoma_foi.tsv' <br> 'fusion_summary_lgat_foi.tsv' <br> fusion_summary_ewings_foi.tsv' <br> 'biospecimen_id_to_bed_map.txt'					| Performs QC on data pre-release files with requirements which should pass before hand off between BIXU Engineering team to the OpenPedCan team		| Yes					| 'data-pre-release-qc.nb.html'						| N/A					| No		| N/A		| N/A | 
```

Reviewer: trying to add a space after the pipe to see if this fixes the table

> 
>  
> 

Reviewer: ```suggestion
```

> | [molecular-subtyping-pathology](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/molecular-subtyping-pathology)                 | `analyses/molecular-subtyping-CRANIO/results/CRANIO_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-EPN/results/CRANIO_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-MB/results/MB_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-neurocytoma/results/neurocytoma_subtyping.tsv` <br> `analyses/molecular-subtyping-EWS/results/EWS_samples.tsv` <br> `analyses/molecular-subtyping-HGG/results/HGG_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-LGAT/results/lgat_subtyping.tsv` <br> `analyses/molecular-subtyping-embryonal/results/embryonal_tumor_molecular_subtypes.tsv` <br>                                                                                                                                                                                                                                                                                                      | Compile output from other molecular subtyping modules and incorporate pathology feedback [#645](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/645)                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | No                               | `choroid_plexus_papilloma_subtypes.tsv` <br>
> `cns-lymphoma-subtypes.tsv` <br>
> `compiled_molecular_subtypes.tsv` <br>
> `compiled_molecular_subtypes_and_report_info.tsv` <br>
> `compiled_molecular_subtypes_with_clinical_feedback_and_report_info.tsv` <br>
> `compiled_molecular_subtypes_with_clinical_pathology_feedback_and_report_info.tsv` <br>
> `cranio_adam_subtypes.tsv` <br>
> `glialneuronal_tumor_subtypes.tsv` <br>
> `juvenile-xanthogranuloma-subtypes.tsv` <br>
> `lgat-pathology-free-text-subtypes.tsv` <br>
> `meningioma_subtypes.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | Yes              | Yes              | GitHub                                                                                                      | N/A                          |

Reviewer: ```suggestion
| [molecular-subtyping-pathology](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/molecular-subtyping-pathology)                 | `analyses/molecular-subtyping-CRANIO/results/CRANIO_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-EPN/results/CRANIO_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-MB/results/MB_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-neurocytoma/results/neurocytoma_subtyping.tsv` <br> `analyses/molecular-subtyping-EWS/results/EWS_samples.tsv` <br> `analyses/molecular-subtyping-HGG/results/HGG_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-LGAT/results/lgat_subtyping.tsv` <br> `analyses/molecular-subtyping-embryonal/results/embryonal_tumor_molecular_subtypes.tsv` <br>                                                                                                                                                                                                                                                                                                      | Compile output from other molecular subtyping modules and incorporate pathology feedback [#645](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/645)                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | No                               | `choroid_plexus_papilloma_subtypes.tsv` <br> `cns-lymphoma-subtypes.tsv` <br> `compiled_molecular_subtypes.tsv` <br>`compiled_molecular_subtypes_and_report_info.tsv` <br> `compiled_molecular_subtypes_with_clinical_feedback_and_report_info.tsv` <br> `compiled_molecular_subtypes_with_clinical_pathology_feedback_and_report_info.tsv` <br> `cranio_adam_subtypes.tsv` <br> `glialneuronal_tumor_subtypes.tsv` <br> `juvenile-xanthogranuloma-subtypes.tsv` <br> `lgat-pathology-free-text-subtypes.tsv` <br>`meningioma_subtypes.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Yes              | Yes              | GitHub                                                                                                      | N/A                          |
```

> | [molecular-subtyping-pathology](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/molecular-subtyping-pathology)                 | `analyses/molecular-subtyping-CRANIO/results/CRANIO_molecular_subtype.tsv` `analyses/molecular-subtyping-EPN/results/CRANIO_molecular_subtype.tsv` `analyses/molecular-subtyping-MB/results/MB_molecular_subtype.tsv` `analyses/molecular-subtyping-neurocytoma/results/neurocytoma_subtyping.tsv` `analyses/molecular-subtyping-EWS/results/EWS_samples.tsv` `analyses/molecular-subtyping-HGG/results/HGG_molecular_subtype.tsv` `analyses/molecular-subtyping-LGAT/results/lgat_subtyping.tsv` `analyses/molecular-subtyping-embryonal/results/embryonal_tumor_molecular_subtypes.tsv`                                                                                                                                                                                                                                                                                                       | Compile output from other molecular subtyping modules and incorporate pathology feedback [#645](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/645)                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | No                               | `results/compiled_molecular_subtyping_with_clinical_feedback.tsv` `results/compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Yes              | Yes              | GitHub                                                                                                      | N/A                          |

Reviewer: ```suggestion
| [molecular-subtyping-pathology](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/molecular-subtyping-pathology)                 | `analyses/molecular-subtyping-CRANIO/results/CRANIO_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-EPN/results/CRANIO_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-MB/results/MB_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-neurocytoma/results/neurocytoma_subtyping.tsv` <br> `analyses/molecular-subtyping-EWS/results/EWS_samples.tsv` <br> `analyses/molecular-subtyping-HGG/results/HGG_molecular_subtype.tsv` <br> `analyses/molecular-subtyping-LGAT/results/lgat_subtyping.tsv` <br> `analyses/molecular-subtyping-embryonal/results/embryonal_tumor_molecular_subtypes.tsv` <br>                                                                                                                                                                                                                                                                                                      | Compile output from other molecular subtyping modules and incorporate pathology feedback [#645](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/645)                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | No                               | `choroid_plexus_papilloma_subtypes.tsv` <br>
`cns-lymphoma-subtypes.tsv` <br>
`compiled_molecular_subtypes.tsv` <br>
`compiled_molecular_subtypes_and_report_info.tsv` <br>
`compiled_molecular_subtypes_with_clinical_feedback_and_report_info.tsv` <br>
`compiled_molecular_subtypes_with_clinical_pathology_feedback_and_report_info.tsv` <br>
`cranio_adam_subtypes.tsv` <br>
`glialneuronal_tumor_subtypes.tsv` <br>
`juvenile-xanthogranuloma-subtypes.tsv` <br>
`lgat-pathology-free-text-subtypes.tsv` <br>
`meningioma_subtypes.tsv`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | Yes              | Yes              | GitHub                                                                                                      | N/A                          |
```

| [mtp-annotations](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/mtp-annotations)					| `scratch/mtp-json/targets/` <br> `scratch/mtp-json/diseases/`					| This module transforms OpenTargetsPlatform Target (core annotations for targets) and Disease/Phenotype (core annotations for diseases and phenotypes) tables into mapping files utilized in filtering MTP designated tables and OPC data release files for plotting API development					| No			|N/A 					| No			 | local			| N/A			| N/A 	|

> | [mtp-tables-qc-checks](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/mtp-tables-qc-checks)					| `gene-level-cnv-consensus-annotated-mut-freq.tsv.gz' 'gene-level-snv-consensus-annotated-mut-freq.tsv.gz` <br> `gene-variant-snv-consensus-annotated-mut-freq.tsv.gz' 'putative-oncogene-gused-gene-freq.tsv.gz` <br> `putative-oncogene-fusion-freq.tsv.gz' 'long_n_tpm_mean_sd_quantitle_gene_wise_zscore.tsv.gz` <br> `long_n_tpm_mean_sd_quatile_group_wise_zscore.tsv.gz`				| Performs summary and QC checks comparing the current and the previous OPC mutation frequencies table					| No			|N/A				| No				| No				| N/A				| N/A				| N/A|

Reviewer: ```suggestion
| [mtp-tables-qc-checks](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/mtp-tables-qc-checks)					| `gene-level-cnv-consensus-annotated-mut-freq.tsv.gz` <br> `gene-level-snv-consensus-annotated-mut-freq.tsv.gz` <br> `gene-variant-snv-consensus-annotated-mut-freq.tsv.gz` `putative-oncogene-gused-gene-freq.tsv.gz` <br> `putative-oncogene-fusion-freq.tsv.gz` `long_n_tpm_mean_sd_quantitle_gene_wise_zscore.tsv.gz` <br> `long_n_tpm_mean_sd_quatile_group_wise_zscore.tsv.gz`				| Performs summary and QC checks comparing the current and the previous OPC mutation frequencies table					| No			|N/A				| No				| No				| N/A				| N/A				| N/A|
```

| [mtp-annotations](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/mtp-annotations)					| 'scratch/mtp-json/targets/' <br> 'scratch/mtp-json/diseases/'					| This module transofroms OpenTargetsPlatform Target (core annotations for targets) and Disease/Phenotype (core annotations for diseases and phenotypes) tables into mapping files utilized in filtering MTP disgenated tables and OPC data release files for ploting API development					| No			|N/A 					| No			 | local			| N/A			| N/A 	|

| [mtp-tables-qc-checks](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/mtp-tables-qc-checks)					|'gene-level-cnv-consensus-annotated-mut-freq.tsv.gz' 'gene-level-snv-consensus-annotated-mut-freq.tsv.gz' 'gene-variant-snv-consensus-annotated-mut-freq.tsv.gz' 'putative-oncogene-gused-gene-freq.tsv.gz' 'putative-oncogene-fusion-freq.tsv.gz' 'long_n_tpm_mean_sd_quantitle_gene_wise_zscore.tsv.gz' 'long_n_tpm_mean_sd_quatile_group_wise_zscore.tsv.gz'				| Performs summary and QC checks comparing the current and the previous OPC mutation frequencies table					| No			|N/A				|No				|No				| N/A				| N/A				| N/A|

> 

Reviewer: ```suggestion
| [mtp-annotations](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/mtp-annotations)					| `scratch/mtp-json/targets/` <br> `scratch/mtp-json/diseases/`					| This module transforms OpenTargetsPlatform Target (core annotations for targets) and Disease/Phenotype (core annotations for diseases and phenotypes) tables into mapping files utilized in filtering MTP designated tables and OPC data release files for plotting API development					| No			|N/A 					| No			 | local			| N/A			| N/A 	|
| [mtp-tables-qc-checks](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/mtp-tables-qc-checks)					| `gene-level-cnv-consensus-annotated-mut-freq.tsv.gz' 'gene-level-snv-consensus-annotated-mut-freq.tsv.gz` <br> `gene-variant-snv-consensus-annotated-mut-freq.tsv.gz' 'putative-oncogene-gused-gene-freq.tsv.gz` <br> `putative-oncogene-fusion-freq.tsv.gz' 'long_n_tpm_mean_sd_quantitle_gene_wise_zscore.tsv.gz` <br> `long_n_tpm_mean_sd_quatile_group_wise_zscore.tsv.gz`				| Performs summary and QC checks comparing the current and the previous OPC mutation frequencies table					| No			|N/A				| No				| No				| N/A				| N/A				| N/A|
```
